lib/pos_snapshot_entries.go

```diff
@@ -257,6 +257,20 @@ func (bav *UtxoView) GetSnapshotValidatorSetEntryByPKID(pkid *PKID) (*ValidatorE
 	return validatorEntry, nil
 }
 
+func (bav *UtxoView) GetFullSnapshotValidatorSet() ([]*ValidatorEntry, error) {
+	snapshotGlobalParamsEntry, err := bav.GetSnapshotGlobalParamsEntry()
+	if err != nil {
+		return nil, errors.Wrapf(err, "GetFullSnapshotValidatorSet: problem retrieving SnapshotGlobalParamsEntry: ")
+	}
+
+	validatorSet, err := bav.GetSnapshotValidatorSetByStakeAmount(snapshotGlobalParamsEntry.ValidatorSetMaxNumValidators)
+	if err != nil {
+		return nil, errors.Wrapf(err, "GetFullSnapshotValidatorSet: problem retrieving ValidatorSetAtTipBlock: ")
+	}
+
+	return validatorSet, nil
+}
```

Comment:
I have an identical function defined in my stack. We'll just want to use one or the other. Mine is named `GetAllSnapshotValidatorSetEntriesByStake` for reference.

Reply:
I'll delete this function once we merge with your feature branch.